src/main/java/uk/gov/justice/hmpps/prison/api/model/RequestToUpdateOffenderDates.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@ApiModel(description = "Update Offender Dates Request")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class RequestToUpdateOffenderDates {
+
+    @ApiModelProperty(required = true, value = "Username who used CRD service to submit calculated dates.")
+    private String submissionUser;
```

Comment:
Decided to include the user in the request payload, as the API which invokes `offender-dates` would be the service user, for example the CRD API, and will not contain the username of the person who submitted the request